### DISCUSSION TITLE
cmake install

### DISCUSSION THREAD

**Discussion by dimavs** [NVIDIA/stdexec#1437] [General]
I am not great with cmake, so I am not sure if it's me or there is a bug.
I am just trying to build like that
`$ cmake -S . -B build`
`$ cmake --build build/ --target install`

which fails with the next error:
>CMake Error at cmake_install.cmake:70 (file):
> file INSTALL cannot find "<< full path >>/libsystem_context.a": No such file or directory.

**Comment by ericniebler** [comment]
are you on windows, linux, or mac?
